buildSrc/src/main/java/com/google/firebase/gradle/plugins/license/ThirdPartyLicensesExtension.java

```diff
@@ -27,18 +28,25 @@ public class ThirdPartyLicensesExtension {
    * supported
    */
   public void add(String name, String... licenseUris) {
-    customLicenses.add(new CustomLicense(name, licenseUris));
+    customLicenses.add(CustomLicense.buildFromStrings(name, licenseUris));
   }
 
   static class CustomLicense implements Serializable {
     final String name;
-    final List<URI> licenseUris = new ArrayList<>();
+    final List<URI> licenseUris;
 
-    CustomLicense(String name, String[] licenseUris) {
+    CustomLicense(String name, List<URI> licenseUris) {
       this.name = name;
+      this.licenseUris = licenseUris;
+    }
+
+    static CustomLicense buildFromStrings(String name, String[] licenseUris) {
+      List<URI> uris = new ArrayList<>();
       for (String s : licenseUris) {
-        this.licenseUris.add(URI.create(s));
+        File file = new File(s);
```

Comment:
afaicr we used to allow specifying `http(s)` urls in `thirdPartyLicenses{}`, looks like it will no longer work.

I think it's fine since it has never been used, but wanted to call it out for future reference.